path_2024.py

```diff
@@ -0,0 +1,76 @@
+"""
+Task 1 path.
+"""
+import pathlib
+import time
+
+import dronekit
+
+from modules import add_takeoff_and_landing_command
+from modules import upload_commands
+from modules import waypoints_to_commands
+from modules import waypoint_tracking
+from modules import waypoint_names_to_coordinates
+from modules import load_waypoint_name_to_coordinates_map
+
+WAYPOINT_FILE_PATH = pathlib.Path(".", "2024", "waypoints_task_2.csv")
+ALTITUDE = 40
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+DELAY = 0.1  # seconds
+
+def run() -> int:
+    """
+    Reads in hardcoded waypoints from csv file and sends drone commands.
+    """
+    
+    # Wait ready is false as the drone may be on the ground
+    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = False)
+
+    # Read in hardcoded waypoints from csv files
+    # Waypoints must be in alphabetical order
```

Comment:
I misunderstood the Python dictionary. The elements are in insertion order, so the alphabetical order isn't necessary. Put that in as the comment instead.

However, we will still want to keep the waypoint names in alphabetical order (so that the order in the CSV file is clear (but you don't need to add this as a comment)).